api/src/controllers/put/inspection.js

```diff
@@ -74,6 +70,20 @@ exports.editMaster = async function (args, res, next, incomingObj) {
     }
   }
 
+  // Flavours:
+  // When editing, we might get a request to make a brand new flavour rather than edit.
+  var observables = [];
+  if (incomingObj.InspectionLNG && incomingObj.InspectionLNG._id) {
+    observables.push(this.editLNG(args, res, next, incomingObj.InspectionLNG)) && delete incomingObj.InspectionLNG;
```

Comment:
Is there any reason why these 2 commands are being chained?

Because observables.push() is never going to fail, so its always going to run the delete anyways.  Unless I am missing something, they might as well just be separate lines, no?
```
if (...) {
  observables.push(this.editLNG(args, res, next, incomingObj.InspectionLNG));
  delete incomingObj.InspectionLNG;
}
```

Same comment for the other instances in the PR.